model_server_tools/model_packaging/export_model.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""
+Command line interface to export model files to be used for inference by MXNet Model Server
+"""
+
+import logging
+from model_server_tools.model_packaging.arg_parser import ArgParser
+from model_server_tools.model_packaging.export_model_utils import ModelExportUtils
+
+
+def export_model(args, manifest, export_file_path=None):
+    """
+    Internal helper for the exporting model command line interface.
+    """
+    model_path = args.model_path
+    model_name = args.model_name
+    temp_files = []
+    try:
+        # Step 1 : Check if .mar already exists with the given model name
```

Comment:
check if model_name contains special chars

Reply:
Will do it.

Reply:
Done in a46fa5d65e8493d56fda684f0cca7f0e97d7cf5c